core/src/main/java/io/grpc/internal/ClientTransport.java

```diff
@@ -136,12 +136,12 @@ ClientStream newStream(MethodDescriptor<?, ?> method,
 
     /**
      * Invoked when a ping is acknowledged. The given argument is the round-trip time of the ping,
-     * in microseconds.
+     * in nanoseconds.
      *
-     * @param roundTripTimeMicros the round-trip duration between the ping being sent and the
+     * @param roundTripTimeNanos the round-trip duration between the ping being sent and the
      *     acknowledgement received
      */
-    void pingAcknowledged(long roundTripTimeMicros);
+    void pingAcknowledged(long roundTripTimeNanos);
```

Comment:
Backwards incompatibility means we are free to break the API, but such silent semantic change is unsettling to me.